content/projects/jcasc/index.adoc

```diff
@@ -0,0 +1,33 @@
+---
+layout: project
+title: "Jenkins Configuration as Code"
+section: projects
+tags:
+- jenkins-configuration-as-code
+- jcasc
+
+---
+
+Configure your Jenkins via human-readable declarative configuration files.
+
+---
+
+image:/images/jcasc-logo.png["Jenkins Configuration as Code"]
```

Comment:
Image has not been published yet. I would also advice to avoid keeping the image on the top page

Reply:
done!